geomstats/geometry/special_orthogonal.py

```diff
@@ -335,6 +336,56 @@ def vector_from_skew_matrix(self, skew_mat):
         """
         return SkewSymmetricMatrices(self.n).basis_representation(skew_mat)
 
+    def to_tangent(self, vector, base_point=None):
+        return self.regularize_tangent_vec(vector, base_point)
+
+    @geomstats.vectorization.decorator(
+        ['else', 'vector', 'else', 'output_point'])
+    def regularize_tangent_vec_at_identity(
```

Comment:
Hm I know this is not your code, but I don't remember the purpose of `regularize`, `regularize_tangent_vec_at_identity` and `regularize_tangent_vec`: is it expected that they all call `regularize`?

Reply:
In vector representation, points are in fact tangent vectors so there's no surprise all these methods are the same.